doc/using_azure-iot-sdk-c_with_the_latest_tls_version_only.md

```diff
@@ -0,0 +1,52 @@
+# How to configure the Azure IoT C SDK and currently supported TLS platforms to use TLS 1.2 only 
```

Comment:
How to configure the Azure IoT C SDK TLS platforms to disable TLS 1.0/1.1

Reply:
Done.

Reply:
I agree - title is not in-line with what some of the techniques describe.
I believe it is critical to allow clients to support TLS1.2+ (i.e. TLS1.3 and 1.2 today) and not force TLS1.2 _only_.